### PR TITLE
Make owl tamable, with compatibility and right code checks

### PR DESCRIPTION
**Pull request by mckaygerhard**
this overrides the made at https://github.com/minetest-mobs-mods/dmobs/pull/7

* cherry picked from https://github.com/BlockySurvival/mobs_animal_pack/pull/6
* not change namespace neither idents

hi @taikedz  please i fixed this pull request .. close the #7  and merge this!

### CODE REVIEW

**Comment by mckaygerhard** [comment]
tested and working (i plan to change the namespace in my own fork with different branch)

screenshot_20220121_124338

File: mobs/owl.lua [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+end
+if minetest.get_modpath("mobs_better_rat") then
+	table.insert(follow, "mobs_better_rat:rat")
+end
+if minetest.get_modpath("mobs_bat") then
+	table.insert(follow, "mobs_bat:bat")
+end
+if minetest.get_modpath("mobs_birds") then
+	table.insert(follow, "mobs_birds:bird_sm")
+end
```
Comment (taikedz) [inline mobs/owl.lua:41]: Unnecessary complication. A simple table (as it was before in the older code) listing the entity names to follow is sufficient, there is no need to actively check.

**Review by taikedz — CHANGES_REQUESTED** [review]
Nearly! It's bad form to add the checks, simply listing entity names is sufficient. `mobs_redo` just checks to see if any in-game entities match one of the names.

File: mobs/owl.lua (hunk 1 above)
Comment (mckaygerhard) [reply mobs/owl.lua:41]: i'll close this and use only the #9 that includes this PR so.. then you arged that the checks are not necesarry due the mobs api take care?  i will remove the checks then

**Comment by mckaygerhard** [comment]
close in flavour of #9 due was included

**Pull request closed without merging**